app/presenters/document/field_group_presenter.rb

```diff
@@ -33,7 +33,8 @@ def display
 
       {
         title: display_title,
-        sections: display_sections
+        sections: display_sections,
+        section_id: @id
```

Comment:
Having two properties `sections` and `section_id` is confusing. Shouldn't the latter be named `group_id` or even just `id`?